crates/object_detection/src/pose_detection.rs

```diff
@@ -268,6 +255,8 @@ fn non_maximum_suppression(
     poses
 }
 
+// Allowing dead code for future use of `scratchpad` once it is restored.
+#[allow(dead_code)]
```

Comment:
please remove instead. We can add this back (if needed)

Reply:
Done :)